ultraplot/axes/base.py

```diff
@@ -2454,6 +2454,39 @@ def _range_tightbbox(self, s):
         else:
             return bbox.ymin, bbox.ymax
 
+    def _unshare(self, *, which: str):
+        """
+        Remove this Axes from the shared Grouper for the given axis ('x', 'y', 'z', or 'view').
+        Note this isolates the axis and does not preserve the transitivity of sharing.
+        """
+        if which not in self._shared_axes:
+            warnings._warn_ultraplot(f"Axis {which} is not shared")
+            return
+        if which in "xy":
+            setattr(self, f"_share{which}", None)  # essential
+        # Note _scale is also set when calling sharex or y.
+        # I think it is fine to leave it as otherwise we would
+        # need to determine the scale, which may get messy.
+
+        grouper = self._shared_axes[which]
+        siblings = list(grouper.get_siblings(self))
+        for sibling in siblings:
+            if sibling is not self:
+                # Unshare by removing them from the grouper
+                grouper.remove(sibling)
+                sibling._shared_axes[which].remove(self)
+                # To be safe let's remove this
+                self._shared_axes[which].remove(sibling)
+                if which in "xy":
+                    setattr(sibling, f"_share{which}", None)
+                this_ax = getattr(self, f"{which}axis")
+                sib_ax = getattr(sibling, f"{which}axis")
+                # Reset formatters
+                this_ax.set_major_locator(mticker.AutoLocator())
+                this_ax.set_major_formatter(pticker.AutoFormatter())
+                this_ax.set_minor_locator(mticker.AutoLocator())
+                this_ax.set_minor_formatter(pticker.AutoFormatter())
```

Comment:
We should make a deepcopy of the current formatter for the axis here instead? 

IIUIC, that way any custom changes to the formatting would be preserved?

Reply:
Ah right. I believe I had this originally but moved this to auto. I think I ran into some snag but will reset it to see if it works.

Reply:
Your suggestion is the way to go I think.

Reply:
Fixed in a latest commit.